Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1304,6 +1304,13 @@ void AXObjectCache::handleChildrenChanged(AccessibilityObject& object)
     else if (auto* axRow = dynamicDowncast<AccessibilityTableRow>(object)) {
         if (auto* parentTable = axRow->parentTable())
             deferRecomputeTableCellSlots(*parentTable);
+    } else if (auto* scrollView = dynamicDowncast<AccessibilityScrollView>(object)) {
+        // When we get a childrenChanged for a iframe (a ScrollView that is not the root web area), then we need to
```

Comment:
typo: "an iframe"

I feel like "a childrenChanged" is a bit confusing of a noun. Maybe we could say something like "When the children of an iframe change, e.g. because it became visible or non-visible, ...